### ISSUE TITLE
xkbcommon/xkbcommon-x11.h: No such file or directory

### ISSUE THREAD

**Issue by baiyuxiong** [go-vgo/robotgo#403]
````
$ go run main.go 
# github.com/robotn/gohook
In file included from ./event/pub.h:20,
                 from ./event/goEvent.h:18,
                 from ../../go/pkg/mod/github.com/robotn/gohook@v0.31.2/hook.go:22:
./event/../hook/x11/input_c.h:40:10: fatal error: xkbcommon/xkbcommon-x11.h: No such file or directory
 #include <xkbcommon/xkbcommon-x11.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
````

- Robotgo version (or commit ref): github.com/go-vgo/robotgo v0.100.8
- Go version: go version go1.17.2 linux/amd64
- Gcc version: gcc (Uos 8.3.0.7-1+dde) 8.3.0
- Operating system and bit: uos 1043 amd64
```Go
package main

import (
	"github.com/go-vgo/robotgo"
)

func main() {
	robotgo.Move(800, 320)
	robotgo.Click("left", true)
	robotgo.TypeStr("Hello World. Winter is coming!")
	robotgo.Move(1200, 320)
	robotgo.Click("left", true)
}

```

**Comment by baiyuxiong** [comment]
sudo apt-get install libxkbcommon-x11-dev

**Comment by vcaesar** [comment]
But there can't reproduce in the Ubuntu.

**Comment by srini-rg** [comment]
I ran into this problem too, on my OpenSuse Tumbleweed. After spending 6 hours, I solved it by chance, when I found that `xkbcommon` directory is *not* created in `/usr/include`, as it should have been, but in `/usr/include/libxkbcommon`.  So, the solution is to create a softlink in `/usr/include` with the command  
```
sudo ln -s /usr/include/libxkbcommon/xkbcommon /usr/include/ 
```
After making this change, I am able to run the [sample programs given](https://github.com/robotn/gohook#examples)

This is the [github link](https://github.com/espanso/espanso/issues/995) where I found the solution. 
@vcaesar , thank you for the wonderful package you have donated to the community! Great Work! Keep it up!
